src/model/data.types.model.ts

```diff
@@ -63,3 +63,7 @@ export const NonLegalFirmNoc = "non_legal_firm_members_nature_of_control_types";
 export const PaymentKey = "payment";
 export const OverseasEntityKey = "overseas_entity_id";
 export const Transactionkey = "transaction_id";
+
+// Is secure and Has sold land Keys
```

Comment:
Comment doesn't add a lot of info that's not immediately obvious from your well named constant vars...

Reply:
Thanks @mwestacott, it seems good to me, have you got any suggestion?

Reply:
Well, my idea was to just remove the comment, but I now see that we've got comments for all these vars, so happy for it to remain (I'm guessing they're arranged in some grouping of logical blocks).